### PR TITLE
.get(Namespace.named(u"monkeys"))

### LINKED ISSUE #14: Support getting a single namespace
#2 introduced namespace creation and enumeration.  Add an API which allows a single namespace to be retrieved by name.

### PR DESCRIPTION
**Pull request by exarkun**
Fixes #14

### CODE REVIEW

File: src/txkube/_network.py [hunk 1]
```diff
@@ -113,10 +113,31 @@ def create(self, obj):
...
+            d.addCallback(log_result, action)
+            d.addCallback(object_from_raw)
+            return d.addActionFinish()
+
+
+    def get(self, obj):
+        """
+        Issue a I{GET} to retrieve the given object.
+
+        The object must have identifying metadata such as a namespace and a
```
Comment (markrwilliams) [inline src/txkube/_network.py:125]: It might be nice to describe the `object` as an `IObject` provider.

File: src/txkube/_network.py [hunk 2]
```diff
@@ -224,6 +245,11 @@ def __repr__(self):
 
 
 
+def log_result(document, action):
```
Comment (markrwilliams) [inline src/txkube/_network.py]: This is pretty self explanatory, but a docstring that made it clear this was for use with eliot might be nice.

**Review by markrwilliams — APPROVED** [review]
@exarkun provided some background:

This allows one to access namespaces as first-class objects.  A followup PR will allow one to access namespaced objects.

The new get method on `IKubernetesClient` won't have to change because `object_location` already knows how to address namespaced objects.

The code is fine and codecov.io says it's 100% covered.  I don't know a lot about Kubernetes so while I approve of these changes it's a weak approval.

File: src/txkube/_network.py (hunk 1 above)
Comment (exarkun) [reply src/txkube/_network.py:125]: Zoops!  Docs, indeed.  In fact, there's an explicit interface off to the side somewhere that I also forgot to update.  Thanks. :)

**Pull request merged**